app/api/orders.py

```diff
@@ -320,7 +317,7 @@ def before_update_object(self, order, data, view_kwargs):
         :return:
         """
         if data.get('amount') or data.get('is_billing_enabled'):
-            check_billing_info(data)
+            check_billing_info(data, order)
```

Comment:
This check won't get executed if I send `is_billing_enabled` as false

Reply:
Yes, Isn't that's what we want. If `is_billing_enabled` is set to false by the organiser, Than its optional for the user to enter or not enter the billing data.

Reply:
This is sent by user...